Formula/kontena-cli.rb

```diff
@@ -0,0 +1,185 @@
+    sha256 "87d81d2d1780de5beb8c25848b027ddcdec7fb407d766aafde0c4f032ccf36c8"
+  end
+
+  resource "unicode-display_width" do
+    url "https://rubygems.org/gems/unicode-display_width-1.1.3.gem"
+    sha256 "1d6247805b2431fd3d27614f6e0c38665b068bfc3a58cf89f08b7f107daf48e4"
+  end
+
+  resource "unicode_utils" do
+    url "https://rubygems.org/gems/unicode_utils-1.4.0.gem"
+    sha256 "b922d0cf2313b6b7136ada6645ce7154ffc86418ca07d53b058efe9eb72f2a40"
+  end
+
+  resource "verse" do
+    url "https://rubygems.org/gems/verse-0.5.0.gem"
+    sha256 "4c8af6322b91e115a772af1acdf99952e391ad14890577674e11e52d474af2e0"
+  end
+
+  resource "websocket-driver-kontena" do
+    url "https://rubygems.org/gems/websocket-driver-kontena-0.6.5.gem"
+    sha256 "7615511da99318dd8131de01587066416c36b7a630c9a2bd70ad3f987fbc2d3f"
+  end
+
+  resource "websocket-extensions" do
+    url "https://rubygems.org/gems/websocket-extensions-0.1.2.gem"
+    sha256 "7919c0310edda55ce4026d3f160e612160493c048276cabe500308439dd2161f"
+  end
+
+  resource "wisper" do
+    url "https://rubygems.org/gems/wisper-1.6.1.gem"
+    sha256 "5ae88a4aa8cab54ec849a934c4e70aebbef560114dd164dfd9b0058ec3077ca5"
+  end
+
+  resource "kontena-plugin-cloud" do
+    url "https://rubygems.org/gems/kontena-plugin-cloud-1.1.0.gem"
+    sha256 "87e88329c17c6aa6c49069fd6fa5f6577c451f73bb2bf264051b858af8ea57f1"
+  end
+
+  def install
+    ENV["GEM_HOME"] = libexec
+
+    cd "cli" do
+      system "gem", "build", "--norc", "kontena-cli.gemspec"
+
+      if build.head?
+        system "gem", "install", Dir["kontena-cli-*.gem"].first,
+         "--no-document", "--norc", "--install-dir", libexec
+      else
+        resources.each do |r|
+          r.verify_download_integrity(r.fetch)
+          system "gem", "install", r.cached_download, "--ignore-dependencies",
+                 "--no-document", "--norc", "--install-dir", libexec
+        end
+
+        system "gem", "install", "--ignore-dependencies",
+         "--norc", "--no-document", "kontena-cli-#{version}.gem"
+      end
+    end
+
+    bin.env_script_all_files(libexec/"bin", :GEM_HOME => ENV["GEM_HOME"])
+
+    (bin/"kontena").write <<-EOS.undent
+      #!/bin/bash
+
+      export GEM_PATH=#{libexec}
+      export KONTENA_EXTRA_BUILDTAGS=homebrew#{",head" if build.head?}
+      exec #{libexec/"bin/kontena"} "$@"
+    EOS
+
+    zsh_completion.install "cli/lib/kontena/scripts/kontena.zsh" => "_kontena"
+    bash_completion.install "cli/lib/kontena/scripts/kontena.bash" => "kontena"
+  end
+
+  test do
+    assert_match "+homebrew", shell_output("#{bin}/kontena --version")
+    assert_match "login", shell_output("#{bin}/kontena complete kontena master")
+    assert_match "Kontena DigitalOcean plugin", shell_output("#{bin}/kontena plugin search digitalocean")
+    assert_match "description: Sample stack to test Kontena deployments", shell_output("#{bin}/kontena stack registry show kontena/hello-world")
```

Comment:
an intermediate `output` variable will allow you to avoid going passed the eightieth column